### DISCUSSION TITLE
Export typescript type

### DISCUSSION THREAD

**Discussion by casualmatt** [directus/directus#16934] [Archive]
Hello to all, 

when building a big TS app around directus is frustrating to manually update your TS types, especially because it ruin the fast prototyping experience that directus provide on the UI side.

Is possible to introduce a command to export a .d.ts file? 
OpenAPI to d.ts is not really working if you want to use the SDK either, those tools are build to be use with fetch. 

Any suggestion on you are solving this issue?

**Comment by jclaveau** [comment]
This seems to be a duplicate of https://github.com/directus/directus/discussions/5418
